### PR TITLE
Modld 589

### PR DESCRIPTION
**Pull request by PBobylev**

### CODE REVIEW

File: src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java [hunk 1]
```diff
@@ -39,136 +25,80 @@
...
-      .errors(List.of(new Error()
-        .message(e.getMessage())
-        .type(e.getClass().getSimpleName())
-        .code(code.getValue())))
-      .totalRecords(1);
-    return buildResponseEntity(errorResponse, status);
+    return genericServerExceptionMapper.errorResponseEntity(exception);
   }
 
   private static ResponseEntity<ErrorResponse> buildResponseEntity(ErrorResponse errorResponse, HttpStatus status) {
```
Comment (pkjacob) [inline src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java:104]: This method is not used any more.

Comment (PBobylev) [reply src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java]: Fixed, thank you!

File: src/main/resources/ValidationMessages.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+primaryTitleConstraint.message=required_primary_main_title
+lccnPatternConstraint.message=lccn_does_not_match_pattern
```
Comment (pkjacob) [inline src/main/resources/ValidationMessages.properties:2]: Minor -> we can change the code to `invalid_lccn_format`

File: src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java [hunk 3]
```diff
@@ -39,140 +24,80 @@
...
-    logException(exception);
-    return buildResponseEntity(exception, BAD_REQUEST, VALIDATION_ERROR);
+    MissingServletRequestParameterException e) {
+    logException(e);
+    return genericBadRequestMapper.errorResponseEntity(e);
+  }
+
+  @ExceptionHandler(RequestProcessingException.class)
+  public ResponseEntity<ErrorResponse> handleRequestProcessingException(RequestProcessingException exception) {
+    return requestProcessingExceptionMapper.errorResponseEntity(exception);
```
Comment (pkjacob) [inline src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java:94]: Call `logException(exception);` here also.

**Review by pkjacob — APPROVED** [review]

File: src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java (hunk 3 above)
Comment (PBobylev) [reply src/main/java/org/folio/linked/data/controller/advice/ApiExceptionHandler.java:94]: I think we shouldn't do it:
RequestProcessingException is our exception, we throw it manually and we log it together with throwing.
While other exceptions are automatic, that's why we log all of them here.

**Review by askhat-abishev — APPROVED** [review]

**Pull request merged**